app/main/update_office/views.py

```diff
@@ -239,7 +239,15 @@ def form_valid(self, form, **kwargs):
                 alias = model_field.alias if model_field.alias else field_name
                 data[alias] = field_value
 
-        pda.update_office_contact_details(form.firm.firm_id, form.office.firm_office_code, data)
+        try:
+            pda.update_office_contact_details(form.firm.firm_id, form.office.firm_office_code, data)
+        except ProviderDataApiError as e:
+            logger.error(f"Error {e.__class__.__name__} whilst updating office contact details {e}")
+            form.form_errors = getattr(form, "form_errors", [])
+            form.form_errors.append("We couldn’t update the office contact details. Try again later.")
```

Comment:
should we be passing in the office code to the error message so the user knows which office was not updated

Reply:
Also is there any reason we dont use the flask flash method of displaying an error the way we have done elsewhere in the service?